CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ Sharpbrake Changelog
 
 ### master
 
+### [v4.5.0][v4.5.0] (January 12, 2018)
+
+* Updated NLog dependencies
+  ([#82](https://github.com/airbrake/sharpbrake/pull/82))
+* Added internal logging
```

Comment:
This is definitely not worth mentioning.

Reply:
I would like to disagree here. Internal logging (tracing) is quite important when you need to debug some unexpected behavior. This is definitely a feature. I am going to refer to that functionality in this issue https://github.com/airbrake/sharpbrake/issues/77.

Reply:
Yes, but it's internal, that is, for the developers of this project. Imagine an Airbrake customer reading this. How's that helpful to them?

Reply:
It's not for the developer of the Sharpbrake itself - this is for developers who integrate Sharpbrake into their projects but something goes bad, e.g. notices are not sent - how to check what is going on here? They can enable tracing and try to figure out the issue (like missing some configuration options or thread-related issue, etc.).

Reply:
Ok, this does seem to be documenting of the public API. No objections!